### ISSUE TITLE
About training (pseudo label loss & log files)

### ISSUE THREAD

**Issue by sukzoon1234** [imatif17/Prototype-Mean-Teacher#6]
Hello. Thank you for your awesome work.

I have 2 questions that arose while reading your paper and running your code.

1. According to previous works of facebook-research (Meta) ([paper1](https://arxiv.org/abs/2102.09480), [paper2](https://arxiv.org/abs/2111.13216)), they did not apply bbox regression loss (applying only cls loss) for RPN and RoI head respectively when calculating unsupervised loss with pseudo-labels. 
This is because the confidence value does not reflect the bbox coordinate information. 
However, in your work, you are using regression loss together, is there any special reason for this?

2. In the paper, the experiment was conducted with a learning rate of 0.2.
However, in the yaml files, **BASE_LR** is 0.02 on crosscamera and 0.04 on crosstime. What is the real learning rate?

3. Do you have the detectron log files from your training, and if so, could you share them with us? 

Once again, thanks for the great work and code sharing.

**Comment by imatif17** [comment]
Hello,

I checked the code (Line 415 of trainer.py) and we're not using the regression loss (for both RPN and RoI head). Thanks for pointing it out. I will update the arXiv version of the paper. 
The Learning rate used is .02. I think there is a typo in the paper.
Unfortunately, I don't have the log file at the moment.
Thanks for pointing out these things. Much Appreciated!

**Comment by imatif17** [comment]
Btw I haven't tuned or got into details about the regression losses. I think it might be helpful to use it. As it might guide and help the student model to better localization. You can try some values other than 0 and see how it works.
